User Audit/useraudit.py

```diff
@@ -31,27 +30,42 @@ def flatten(x, name='', exclude=exclude):
     return out
 
 token = sys.argv[1]
-base_url = "https://api.kennasecurity.com"
-users_url= base_url + "/users"
+# increase per_page to get more data per request
+per_page = 10
```

Comment:
can this be configurable like user can adjust this number?

Reply:
these scripts are sent to the client so either CX or client can customize the scripts. I have added a small README snippet about `per_page` but I do not see the need to make it a CLI option. I do not like the way CLI is being used in user audit script but not planning to change it